### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by OsvaldoBC**
# Good Day! 🌤️ 
This pull request is intended to create the required classes to represent students and teachers.

## 🔨  Changes made

 ✔️ Create repo.
✔️ Install gemfile and rubocop.
 ✔️ Create a person class.
✔️ Create student and teacher subclasses from person.
✔️ Add readme file

I'm expecting feedback.

### CODE REVIEW

File: teacher.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require './person'
+
+# Class teacher
+class Teacher < Person
+  def initialize(name, age, specialization, parent_permission: true)
```
Comment (elisha2kyakpo1) [inline teacher.rb]: - [ ] The name parameter should have a default value of `Unknown`.

name

File: student.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require './person'
+
+# Class student
+class Student < Person
+  def initialize(name, age, classroom, parent_permission: true)
```
Comment (elisha2kyakpo1) [inline student.rb]: - [ ] The name parameter should have a default value of `Unknown`.

name

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @OsvaldoBC, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @OsvaldoBC    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**